### DISCUSSION TITLE
Why do I get?: [coc.nvim] List files not found

### DISCUSSION THREAD

**Discussion by asheq-svmx** [neoclide/coc.nvim#4772] [Q&A] [answered]
Why do I get these errors?

- On `:CocList files`, I get `[coc.nvim] List files not found`
- On `:CocList buffers`, I get `[coc.nvim] List buffers not found`
- On `:CocList mru`, I get `[coc.nvim] List mru not found`

Thank you!

**Comment by fannheyward** [comment, marked as answer]
You needed to install `coc-lists` extension, `CocInstall coc-lists`. 

https://github.com/neoclide/coc-lists

**Reply by asheq-svmx** [reply]
Thank you!

The sample configuration in https://github.com/neoclide/coc.nvim#readme includes mappings for these sources but don't mention anything about coc-lists.
